### DISCUSSION TITLE
SSO issue

### DISCUSSION THREAD

**Discussion by Ankitcapco** [okd-project/okd#1061] [General]
Hi there, I am running OKD via helm chart. I tried to authenticate cluster via microsoft authenticator but not able to succeed. As per the docs i did my cnfiguration but afterauthenticating successfully OKD page is refreshing again and again. Please help.
Here is conf- - name: BRIDGE_BASE_ADDRESS
              value: https://abc
            - name: BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS
              value: "true" # no tls enabled
            - name: BRIDGE_USER_AUTH_OIDC_ISSUER_URL
              value: https://sts.windows.net/355839ee-fa1f-44d9-b7de-f83955609b52/
            - name: BRIDGE_USER_AUTH_OIDC_CLIENT_ID
              value: *****************************
            - name: BRIDGE_USER_AUTH_OIDC_CLIENT_SECRET
              value: ********************************

**Comment by amarpreet-capco** [comment]
Hi, did anyone had the same issue or did same deployment of authenticating openshift via azure AD
